pages/nyamatvarden.vue

```diff
@@ -94,16 +120,18 @@ export default {
                 'lon': component.posLon,
                 'lat': component.posLat
               },
-              'depth': 0.0
+              'depth': component.slider
             }
           }
         },
         headers: { 'content-type': 'application/json' }
       }).then(
         (result) => {
           console.log(result.data)
```

Comment:
What's the point in logging the result data here? Do we still need it for testing?